src/class-wpml-gutenberg-integration.php

```diff
@@ -257,17 +257,30 @@ private function guess_inner_HTML_parts( $block ) {
 
 		$parts = array( $inner_HTML, '' );
 
-		preg_match( '#>\s*</#', $inner_HTML, $matches );
-
-		if ( count( $matches ) === 1 ) {
-			$parts = explode( $matches[0], $inner_HTML );
-			if ( count( $parts ) === 2 ) {
-				$match_mid_point = 1 + ( strlen( $matches[0] ) - 3 ) / 2;
-				// This is the first ">" char plus half the remaining between the tags
-
-				$parts[0]        .= substr( $matches[0], 0, $match_mid_point );
-				$parts[1]        = substr( $matches[0], $match_mid_point ) . $parts[1];
-			}
+		switch( $block->blockName ) {
+			case 'core/media-text':
```

Comment:
@adrianosferreira, you wrote a test for this case but if you put a breakpoint on this line, you will see that it never stops.

Reply:
@strategio if the multibyte thing is a blocking issue for you I will hold this fix until @BruceOnTheGo is online to check what he thinks.

I will check the test issue.

Reply:
I'll leave the multi-bytes suggestion up to you @adrianosferreira (to not delay our release), not blocking.